styles/style.css

```
@@ -6,5 +6,87 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Para colocar el fondo debías utilizar la imagen bgtile como fondo del body y hacer un pequeño hack.

```suggestion
  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
  margin: 0;
  background-image: url("../images/bgtile.png");
  background-repeat-y: no-repeat;
```
colocas la imagen y el truco está en decirle que *no* se repita en el eje y.